nengo/spa/assoc_mem.py

```diff
@@ -1,11 +1,5 @@
-import numpy as np
-
-import nengo
-from nengo.dists import Choice, Uniform
-from nengo.networks.ensemblearray import EnsembleArray
+from nengo.networks.assoc_mem import AssociativeMemory as AssocMem
```

Comment:
Personally, I would just do `nengo.networks.AssociativeMemory` down below where you actually use this (and make sure the AssociativeMemory network is imported into the `nengo.networks` namespace). It's just a little clearer to me, because nothing gets renamed, and you're just using it one place, so no need for brevity.